crescent/typedefs.py

```diff
@@ -42,28 +43,32 @@
 )
 
 CommandCallbackT = Callable[..., Awaitable[Any]]
-UserCommandCallbackT = Callable[["Context", User], Awaitable[None]]
-MessageCommandCallbackT = Callable[["Context", Message], Awaitable[None]]
+UserCommandCallbackT = Callable[[Any, User], Awaitable[None]]
+MessageCommandCallbackT = Callable[[Any, Message], Awaitable[None]]
```

Comment:
Why not BaseContext? Same for other type vars below

Reply:
If I say a command accepts `BaseContext` that means it accepts `BaseContext` or a superclass. `Any` is the only way to allow `T` or subclass of `T`